templates/server-statefulset.yaml

```diff
@@ -98,6 +104,9 @@ spec:
 
               exec /bin/consul agent \
                 -advertise="${POD_IP}" \
+                {{- if .Values.server.exposeWANPorts }}
+                -advertise-wan="${NODE_IP}" \
```

Comment:
NOTE: needs to also advertise the node_ip on the lan for folks joining external clients (https://github.com/hashicorp/consul-helm/issues/358#issuecomment-594960342)